examples/wearable/UIComponents/contents/presentation-views/list/main-text-only.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		Wearable UI
+	</title>
+	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" rel="stylesheet" />
+	<script src="../../../js/circle-helper.js"></script>
+	<script src="../../../lib/tau/wearable/js/tau.js"></script>
+</head>
+
+<body>
+	<div class="ui-page" id="arc-listview-page">
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Main Text Only
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-arc-listview ui-arc-listview-5" data-visible-items="5">
+				<li>
+					<a href="#">
+						List01
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						List02
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						List03
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						List04
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						List05
+					</a>
+				</li>
+			</ul>
+		</div>
+		<script src="list_arclistview.js"></script>
```

Comment:
Our file name rules are kebab.
Would you change it to "list-arclistview.js",please?

Reply:
Ok, changed.